src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+    private LocalDate deadline;
+
+    /**
+     * Task with a specified deadline.
+     * @param description description of task.
+     * @param deadline deadline of task.
+     */
+    public Deadline(String description, String deadline) {
+        super(description);
+        try {
+            this.deadline = LocalDate.parse(deadline);
+        } catch (DateTimeParseException e) {
+            this.deadline = null;
+            System.out.println("Deadline was not readable, saving as unspecified deadline");
+        }
+
+    }
+
+    @Override
+    public String toString() {
+        String stem = super.toString();
```

Comment:
Should a more descriptive variable name be used here? I noticed this in the `toString()` method in some other classes (namely Event and Deadline) too.

Reply:
As the variable names are short-lived and apparent I think the use of the name stem is appropriate